docs/iris/src/whatsnew/1.12.rst

```diff
@@ -102,19 +102,6 @@ Iris 1.12 Features
   loaded and returned.
   Previously, this simply resulted in an unrecoverable error.
 
-* Support for "fast" loading of UM files has been added.
-  This can dramatically accelerate loading speeds of fieldsfiles and PP files,
-  if they meet certain common structural requirements, typical for these types
-  of datafile.
-
-  See : :meth:`iris.fileformats.um.structured_um_loading`.
-
-  .. Note:
-
-    This updates and replaces the experimental code in
-    :mod:`iris.experimental.fieldsfile`, which is now deprecated in favour of
-    the new, supported facility.
-
```

Comment:
@pp-mo Only took this out because there is a big section about it higher up, and this bit doesn't really say anything different.  Can put it back in tough, if you are really attached to it.